### PR TITLE
Update to be compatible with 2.0 protocol

### PR DESCRIPTION
**Pull request by adammitchelldev**
## What is the goal of this PR?

The concept API protocol for 2.0 is available, and we need to begin testing integration through the whole stack as soon as we can. This PR aligns client-java with the new protocol, implementing the new methods and using the new terminology.

With this PR, queries are not available.

## What are the changes implemented in this PR?

- Refactored methods to be the same as the 2.0 available concept methods.
- Updated to use the 2.0 concept API

### CODE REVIEW

File: rpc/Transceiver.java [hunk 1]
```diff
@@ -60,46 +60,49 @@
...
+    public static Transceiver create(GraknGrpc.GraknStub stub) {
         ResponseListener responseListener = new ResponseListener();
-        StreamObserver<Transaction.Req> requestSender = stub.transaction(responseListener);
+        StreamObserver<Req> requestSender = stub.transaction(responseListener);
         return new Transceiver(requestSender, responseListener);
     }
 
     /**
      * Send a request and return immediately.
          * This method is non-blocking - it returns immediately.
```
Comment (flyingsilverfin) [inline rpc/Transceiver.java:79]: formatting?

File: test/BUILD [hunk 2]
```diff
@@ -23,3 +23,11 @@ artifact_extractor(
     name = "grakn-extractor",
     artifact = "@graknlabs_grakn_core_artifact//file",
 )
+
+java_test(
```
Comment (flyingsilverfin) [inline test/BUILD:27]: eh?

File: test/Grakn2Test.java [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+import java.util.Objects;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class Grakn2Test {
```
Comment (flyingsilverfin) [inline test/Grakn2Test.java:23]: was this meant to be added?

File: test/integration/concept/ConceptIT.java [hunk 4]
```diff
@@ -502,10 +498,9 @@ public void whenSettingTypeLabel_LabelIsSetToType() {
 
     @Test
     public void whenSettingAndDeletingRelationRelatesRole_RoleInRelationIsSetAndDeleted() {
-        friendship.relates(friend);
         assertTrue(friendship.roles().anyMatch(c -> c.equals(friend)));
 
-        friendship.unrelate(friend);
+        friend.delete();
```
Comment (flyingsilverfin) [inline test/integration/concept/ConceptIT.java:503]: is this the same test??

Comment (adammitchelldev) [reply test/integration/concept/ConceptIT.java:503]: yes

**Pull request closed without merging**